src/hyperparam/one_dimensional_range_methods.jl

```diff
@@ -423,11 +423,11 @@ function Distributions.sampler(r::NominalRange{T,N}) where {T, N}
     return sampler(r, fill(1/N, N))
 end
 
-Base.rand(s::NominalSampler, dims::Integer...) where I<:Integer =
+Base.rand(s::NominalSampler, dims::I...) where I<:Integer =
```

Comment:
Seems to me this could be simplified further, as the concrete subtype `I <: Integer` is not used.  

```suggestion
Base.rand(s::NominalSampler, dims::Integer...) = 
```

Right?

Reply:
Depends on whether you want to allow dims to be heterogeneous (for example, a mix of Int32 and Int64). The syntax I chose disallows that, while your simplified syntax allows it.

Reply:
I was wrong in the first place to make that choice arbitrarily, instead of asking first.

Reply:
> Depends on whether you want to allow dims to be heterogeneous (for example, a mix of Int32 and Int64). 

Fair point. Then this is fine as is.